doc/reference/configuration/index.rst

```diff
@@ -296,66 +300,106 @@ dynamic, that is, they can be changed at runtime by calling ``box.cfg{}``
 a second time.
 
 To see all the non-null parameters, say ``box.cfg`` (no parentheses). To see a
-particular parameter, for example the listen address, say ``box.cfg.listen``.
+particular parameter, for example, the listen address, say ``box.cfg.listen``.
+
+..  _box-cfg-params-prior:
+
+Methods of setting and priorities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tarantool configuration parameters can be specified in different ways.
+The priority of parameter sources is the following, from higher to lower:
+
+*   ``box.cfg{}`` call
```

Comment:
`box.cfg{}` call options, maybe? To make it clear and strictly separated from defaults.

Reply:
Corrected